src/naemon/logging.h

```diff
@@ -39,6 +39,7 @@
 
 #define NSLOG_HOST_NOTIFICATION		524288
 #define NSLOG_SERVICE_NOTIFICATION	1048576
+#define NSLOG_EXT_CUSTOM			2097152
```

Comment:
Can you please remove one `\t` so all numbers line up nicely

Reply:
looks aligned too me with ts=4. But i can replace all tabs in that file with spaces, but that'll be an extra commit.

Reply:
Looks like every editor has it's own opinion on that.
It's sort of aligned on the GitHub Website, but in VS Code the numbers are all over the place.

I guess your suggestion is the best option :)

![grafik](https://github.com/user-attachments/assets/d06b26e6-c751-4885-ae56-aa0448aa2a61)